src/test/java/tutorspet/model/lesson/LessonTest.java

```diff
@@ -93,7 +94,7 @@ public void isSameLesson() {
 
         // different start time -> returns false
         Lesson editedLesson = new LessonBuilder(LESSON_FRI_8_TO_10)
-                .withStartTime(parse(VALID_START_TIME_1400_LESSON_WED_2_TO_4,
+                .withStartTime(parse(VALID_START_TIME_0900,
```

Comment:
Sry, is there a reason why this is being replaced?

Reply:
The end time of the original lesson was 10am, so the start time cannot be set to 2pm. If not I will get a lesson starting at 10 and ending at 2

Reply:
I see!